packages/design-system/src/components/PrivacySettingsDialog/__snapshots__/PrivacySettingsTable.test.jsx.snap

```diff
@@ -398,7 +398,7 @@ Object {
             >
               <legend
                 class="ds-c-label"
-                id="field_2-label"
+                id="choice-list--2__label"
               >
                 <span
```

Comment:
🤔 this empty span doing nothing looks sus (unrelated to your work). should probably be removed in another ticket?

Reply:
I actually think this might be fixed in the hint text updates that are currently in a feature branch